ui/widgets/nuxeo-collapsible.js

```diff
@@ -0,0 +1,134 @@
+import '@polymer/iron-icons/hardware-icons.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+{
+  /**
+   * An element for creating a collapsible block of content.
+   * It comes with a default chevron that can be displayed at the end (default) or at the beggining of the heading.
+   * The heading can be expanded by default.
+   * The chevron has its own slot so it can be also customized.
+   * It is also possible to disable the default animation.
+   *
+   * Example:
+   *
+   *     <nuxeo-collapsible chevron chevronAtStart opened>
+   *        <label slot="heading">Nuxeo</label>
+   *        <div>Collapsed content</div>
+   *     </nuxeo-collapsible>
+   *
+   * ### Styling
+   * The following custom properties and mixins are available for styling:
+   * Custom property | Description | Default
+   * ----------------|-------------|----------
+   * `--nuxeo-collapsible-heading-padding` | Heading padding | `4px`
+   * `--nuxeo-collapsible-icon-height` | Chevron height | `20px`
+   * `--nuxeo-collapsible-icon-width` | Chevron width | `20px`
+   *
+   * @memberof Nuxeo
+   *
+   */
+
+  class Collapsible extends mixinBehaviors([], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          button {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin: 0;
+            padding: var(--nuxeo-collapsible-heading-padding, 4px);
+            background: transparent;
+            border: none;
+            cursor: pointer;
+            text-decoration: none;
+            font-size: 1rem;
+          }
+          /* to fix a blinking default style on safari */
+          button:active {
+            color: inherit;
+          }
+          .heading {
+            width: calc(100% - var(--nuxeo-collapsible-icon-width, 20px));
+            overflow-x: hidden;
+            text-align: start;
+          }
+          iron-icon {
+            --iron-icon-height: var(--nuxeo-collapsible-icon-height, 20px);
+            --iron-icon-width: var(--nuxeo-collapsible-icon-width, 20px);
+          }
+          iron-collapse {
+            padding: 0 var(--nuxeo-collapsible-heading-padding, 4px);
+          }
+        </style>
+
+        <button aria-expanded="[[opened]]" on-tap="_toggle">
+          <slot class="heading" name="heading"></slot>
+          <iron-icon icon="[[_toggleIcon(opened)]]"></iron-icon>
+        </button>
+        <iron-collapse opened="{{opened}}" no-animation="[[!animation]]">
+          <slot></slot>
+        </iron-collapse>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-collapsible';
+    }
+
+    static get properties() {
+      return {
+        /**
+         * Set animation to false to disable animations
+         */
+        animation: {
```

Comment:
Have you tried setting this to false in pure html ? ;)
We can't have boolean properties with default values of true because then we can't set them to false without interacting with the properties. All attributes in HTML are just strings, a boolean property mapped to an attribute is true as long as the attribute is present so you have no way to "unset" this "animation" attribute (that's why the iron-collapse uses a "no-animation")